terraform/README.md

```diff
@@ -0,0 +1,67 @@
+# Deploying with Terraform + Juju + Kubernetes
+
+The `testflinger.tf` plan in this directory can be used for deploying
+the Testflinger server into a dev, staging, or production environment.
+For development, it is recommended to use Juju with Microk8s.
+
+
+## Deploying an Environment
+
+The following assumes that you have an existing Juju + k8s environment
+with a `k8s-certification` credential imported in juju for connecting
```

Comment:
Actually you lost me here, I have not had to do this with microk8s locally, I don't think. What needs to be done on Charmed Kubernetes or microk8s to accomplish this?

Reply:
I can see how that would be confusing. I'll provide operational documentation that is specific to staging or production for things like this, and instead focus this in-repo README on deploying a dev environment.